src/components/filters/MultiFilter.vue

```diff
@@ -0,0 +1,203 @@
+<template>
+  <div>
+    <b-input-group>
+      <b-form-input
+        v-model="query"
+        :name="name"
+        :placeholder="placeholder"
+        trim
+      />
+      <b-input-group-append>
+        <b-button
+          variant="outline-secondary"
+          :disabled="isLoading"
+          @click.prevent="clearFilter"
+        >
+          <font-awesome-icon
+            v-if="isLoading"
+            icon="spinner"
+            class="fa-spin"
+            size="xs"
+          />
+          <font-awesome-icon
+            v-else
+            icon="times"
+          />
+        </b-button>
+      </b-input-group-append>
+    </b-input-group>
+    <font-awesome-icon
+      v-if="foundOptionCount === 100"
+      icon="exclamation-triangle"
+      class="warning text-danger"
+      size="xs"
+      title="There are 100 or more results found, only the first 100 are available. Please refine your search."
+    />
+    <b-form-checkbox-group
+      v-if="slicedOptions.length"
+      v-model="selection"
+      class="checkbox-list"
+      :name="name"
+      :options="slicedOptions"
+      stacked
+    />
+
+    <b-link
+      v-if="showCount < inputOptions.length"
+      class="card-link"
+      @click="showMore"
+    >
+      {{ showMoreText }}
+    </b-link>
+  </div>
+</template>
+
+<script>
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { toRsqlValue } from '@molgenis/rsql'
+import {
+  faSpinner,
+  faTimes,
+  faExclamationTriangle
+} from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+library.add(faTimes)
+library.add(faExclamationTriangle)
```

Comment:
This can go on one line 
library.add(faTimes, faExclamationTriangle, faSpinner)